src/app/directives/strongRoute/strong-route.directive.ts

```diff
@@ -48,10 +52,19 @@ export class StrongRouteDirective
         .subscribe((params) => (this.angularRouteParams = params));
     }
 
-    this.routerLink = this.strongRoute?.toRouterLink({
-      ...this.angularRouteParams,
-      ...this.routeParams,
-    });
+    if (this.config.settings.hideProjects) {
+      this.routerLink = this.strongRoute?.toRouterLink({
+        projectId: Region.defaultProjectId,
+        ...this.angularRouteParams,
+        ...this.routeParams,
+      });
+    } else {
+      this.routerLink = this.strongRoute?.toRouterLink({
+        ...this.angularRouteParams,
+        ...this.routeParams,
+      });
+    }
+
```

Comment:
Ok, this seems like a major violation of separation of concerns. Why does strong route need to know about the project hide setting?

Reply:
I am going to have to insert default project ids into strong routes somewhere in the app, AFAIK there is no alternative to this idea without remaking the menu system to allow components to have multiple menu routes (which'll take too long). This was the optimal place i could find where i can update any links to include a default project id if needed when the `hideProjects` setting is active. If you have an alternative to suggest, i would love to hear

Reply:
I'm not convinced there is a problem. We should probably step through this in a call.

Essentially though:
- load all regions
- regions have a project_id
- when a region is navigated too, use that project id in the route
- everything else should be normal

You may need a new page, a shallow regions list, to kick this process off.